ms-authentication/main.go

```diff
@@ -1,49 +1,47 @@
-// Copyright © 2020 Intel Corporation. All rights reserved.
+// Copyright © 2022 Intel Corporation. All rights reserved.
 // SPDX-License-Identifier: BSD-3-Clause
 
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"ms-authentication/routes"
 
-	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
 )
 
 const (
 	serviceKey = "ms-authentication"
 )
 
-func main() {
-	// Create an instance of the EdgeX SDK and initialize it.
-	edgexSdk := &appsdk.AppFunctionsSDK{ServiceKey: serviceKey}
-	if err := edgexSdk.Initialize(); err != nil {
-		edgexSdk.LoggingClient.Error(fmt.Sprintf("SDK initialization failed: %v\n", err))
-		os.Exit(-1)
-	}
+type AuthApp struct {
+	service interfaces.ApplicationService
+	lc      logger.LoggingClient
+}
```

Comment:
Not needed. Left over from wrapper

Reply:
we still need AuthApp for main to create app

Reply:
main shouldn't be creating app. Doesn't add anything but more complexity. Keep it simple :-)